optparse-applicative.cabal

```diff
@@ -87,19 +87,23 @@ library
                      , Options.Applicative.Common
                      , Options.Applicative.Extra
                      , Options.Applicative.Help
+                     , Options.Applicative.Help.Ann
                      , Options.Applicative.Help.Chunk
                      , Options.Applicative.Help.Core
                      , Options.Applicative.Help.Levenshtein
                      , Options.Applicative.Help.Pretty
+                     , Options.Applicative.Help.Style
                      , Options.Applicative.Help.Types
                      , Options.Applicative.NonEmpty
                      , Options.Applicative.Types
                      , Options.Applicative.Internal
 
   build-depends:       base                            == 4.*
+                     , ansi-terminal                   >= 0.4.0
+                     , text                            >= 1.2
```

Comment:
Any chance to avoid `text` dependency? Otherwise tests/benchmarks inside `text` (and any packages it depends on) become impossible because of a circular dependency. Note that `prettyprinter` has a flag to depend on `base` only.

Reply:
Done!